client/blocks/term-tree-selector/terms.jsx

```diff
@@ -22,6 +22,9 @@ import {
 /**
  * Internal dependencies
  */
+import FormCheckbox from 'components/forms/form-checkbox';
+import FormRadio from 'components/forms/form-radio';
+import FormLabel from 'components/forms/form-label';
```

Comment:
Nit: Alphabetically ordered imports are usually easier to read.

Reply:
Should we enable the ESLint rule for that?

Reply:
I and @sirreal would love that 😜